armory/datasets/package.py

```diff
@@ -0,0 +1,124 @@
+from pathlib import Path
+import os
+import random
+import shutil
+import string
+import subprocess
+
+from armory.logs import log
+from armory.datasets import build, common
+
+
+def package(
+    name,
+    version: str = None,
+    data_dir: str = None,
+    cache_subdir=common.CACHE_SUBDIR,
+    overwrite: bool = False,
+) -> str:
+    """
+    Package a built dataset as .tar.gz, return path
+    """
+    version, data_dir, built_data_dir, subdir = build.build_info(
+        name, version=version, data_dir=data_dir
+    )
+
+    data_dir = Path(data_dir)
+    expected_dir = data_dir / name / version
```

Comment:
equivalently (I think):
```
expected_dir = data_dir / subdir
```

Reply:
correct. I'm not sure if we actually need to return subdir in `build_info`. May be better to factor out, but I'm not sure - there may be datasets with weird pathing.